superset/sql_lab.py

```diff
@@ -190,6 +190,10 @@ def execute_sql_statement(
     db_engine_spec = database.db_engine_spec
     parsed_query = ParsedQuery(sql_statement)
     sql = parsed_query.stripped()
+    if query.limit is None:
+        limited_query = query.limit
+    else:
+        limited_query = query.limit + 1
```

Comment:
The name `limited_query` is a bit confusing because it suggests that this is a query. Maybe use `increased_limit`?

```suggestion
     increased_limit = None if query.limit is None else query.limit + 1
```